src/python/kiss_icp/pipeline.py

```diff
@@ -42,9 +41,7 @@ class OdometryPipeline:
     def __init__(
         self,
         dataset,
-        config: Path,
-        deskew: Optional[bool] = False,
-        max_range: Optional[float] = None,
+        config: KISSConfig,
```

Comment:
This change is in general desire, but I see the disadvantage of always having to build a `KISSConfig` object to use the pipeline. The other arguments are more "standard" which basically means I can fire up a python shell now, and `from kiss_icp.pipeline import Odometrypipeline` and run it as it it.

Would you mind reverting this back to the old bheavior?